<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OrderForGoods</class>
 <widget class="QWidget" name="OrderForGoods">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>913</width>
    <height>562</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>80</y>
     <width>471</width>
     <height>331</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="GoBackBt">
      <property name="styleSheet">
       <string notr="true">border-image: url(:/ICON/picture/Back.ico);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QWidget" name="Bt_widget" native="true">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(240, 240, 240);</string>
        </property>
        <widget class="QWidget" name="horizontalLayoutWidget">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>461</width>
           <height>16</height>
          </rect>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_Bts">
          <item>
           <widget class="QPushButton" name="pushButton_5">
            <property name="styleSheet">
             <string notr="true">border:none;</string>
            </property>
            <property name="text">
             <string>全部</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_4">
            <property name="styleSheet">
             <string notr="true">border:none</string>
            </property>
            <property name="text">
             <string>待付款</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_3">
            <property name="styleSheet">
             <string notr="true">border:none</string>
            </property>
            <property name="text">
             <string>待发货</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_2">
            <property name="styleSheet">
             <string notr="true">border:none</string>
            </property>
            <property name="text">
             <string>待评价</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <widget class="QWidget" name="page"/>
      <widget class="QWidget" name="page_2"/>
     </widget>
    </item>
   </layout>
  </widget>
  <zorder>verticalLayoutWidget</zorder>
  <zorder>Bt_widget</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
